solutions/pytorch_tests/README.md

```diff
@@ -0,0 +1,26 @@
+### 1. test/test_torch.py - Basic tests for PyTorch functionality.
```

Comment:
should this be just test_torch.py instead of test/test_torch.py

Reply:
Fixed.